articles/active-directory-b2c/identity-provider-adfs2016-custom.md

```diff
@@ -194,8 +194,7 @@ Open a browser and navigate to the URL. Make sure you type the correct URL and t
     | E-Mail-Address | email |
     | Display-Name | name |
 
-    Note that these names will not display in the outgoing claim type dropdown. You need to manually type them in. (The dropdown is actually editable).
-
+    Note that these names will not display in the outgoing claim type dropdown. You need to manually type them in. (The dropdown is actually editable). The exception here is Name. It will take the existing one and that will issue the claim with Name Space. Still, that is already covered by the Output claim on the policy. 
```

Comment:
I'd change the first sentence to *that **some of** these names will not display in...*

Could you clarity the second sentence: The exception here is Name. It will take the existing one and that will issue the claim with Name Space. Still, that is already covered by the Output claim on the policy.

Reply:
Yes, I'm trying to explain that the Name claim will fall back to the default one.  I mean that despite the fact that you type "name" it will fall back to the existing "Name" one. Or  not sure if we just say that for "name" just select the one from the drop down menu. That's easier and more direct